authorization/src/test.rs

```diff
@@ -5,18 +5,18 @@ use ed25519_dalek::Keypair;
 use rand::thread_rng;
 use soroban_sdk::testutils::ed25519::Sign;
 
-use soroban_sdk::{BytesN, Env, RawVal, Vec};
+use soroban_sdk::{BytesN, Env, RawVal, Symbol, Vec};
 use soroban_sdk_auth::{Ed25519Signature, SignaturePayload, SignaturePayloadV0};
 
-pub fn to_ed25519(e: &Env, kp: &Keypair) -> Identifier {
-    Identifier::Ed25519(kp.public.to_bytes().into_val(e))
-}
-
 fn generate_keypair() -> Keypair {
     Keypair::generate(&mut thread_rng())
 }
 
-fn make_auth(e: &Env, kp: &Keypair, args: Vec<RawVal>, function: &str) -> Signature {
+fn make_identifier(e: &Env, kp: &Keypair) -> Identifier {
+    Identifier::Ed25519(kp.public.to_bytes().into_val(e))
+}
+
+fn make_auth(e: &Env, kp: &Keypair, function: &str, args: Vec<RawVal>) -> Signature {
```

Comment:
The order of `function` and `args` have been swapped in `make_auth`, so that the order matches the order expected by the `soroban_sdk::check_auth` function.